### PR TITLE
Uyuni 2024.09 remove legacy #25302 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/25302

### CODE REVIEW

**Comment by raulillo82** [comment]
Sections of the screenshot in the related issue are indeed gone. The docs built correctly for English language (checked with uyuni-docs-helper). Sending my approval (but do not merge till we're releasing, and not without more approvals).

**Review by raulillo82 — APPROVED** [review]
LGTM, see previous comment about when to merge.

**Review by 0rnela — APPROVED** [review]

**Pull request merged**
